static/js/vue-cdr-access/src/components/displayWrapper.vue

```diff
@@ -96,17 +106,23 @@ Top level component for full record pages with searching/browsing, including Adm
                 return this.showWidget || this.coerceWorksOnly(this.$route.query.works_only);
             },
 
-
+            minimumCreatedYear() {
```

Comment:
I wonder if we want to move this to mixin that can be used across components?

Reply:
apparently I forgot to even set `min_created_year` to a value, I guess that's one disadvantage to shallowMounting the subcomponents (I tried to regular mount for the displayWrapper tests but it gave me some really weird errors and I backed off after a while). But yeah I can move it over.